addons/flashdrive/functions/fnc_initInterface.sqf

```diff
@@ -11,72 +11,83 @@
 
 params ["_device", "_config"];
 
-private _flash_drives = {
-	params ["_target", "_player", "_params"];
-	_params params ["_config"];
-
-	private _actions = [];
-	private _class = configFile >> "CfgWeapons" >> "Item_FlashDisk_AE3";
-
-	private _add = {
-		params ["_target", "_player", "_params"];
-		([_target, _player] + _params) remoteExec ["AE3_flashdrive_fnc_connectFlashDrive", 2];
+if(!isDedicated) then
+{
+	// get or add parent action
+	private _parentActionPath = _device getVariable ["AE3_parentActionPath", []];
+	if (_parentActionPath isEqualTo []) then
+	{
+		private _parentAction = ["AE3_ParentAction", _name, "", {}, {true}] call ace_interact_menu_fnc_createAction;
```

Comment:
Where is `_name` defined?

Reply:
Nowhere. It's missing. You are right. I need to rewrite some parts.